lib/mvi/service.rb

```diff
@@ -123,7 +127,7 @@ def logging_context(user)
     def create_profile_message(user)
       return message_icn(user) if user.mhv_icn.present? # from SAML::UserAttributes::MHV::BasicLOA3User
       return message_edipi(user) if user.dslogon_edipi.present? && Settings.mvi.edipi_search
-      raise Common::Exceptions::ValidationErrors, user unless user.valid?(:loa3_user)
+      raise Common::Exceptions::ValidationErrors, user unless user.valid?
```

Comment:
hmmm, i think it's important that user validation happen for an loa3_user specifically, those baseline attributes from the saml assertion are really important and only validated if loa3_user

Reply:
I think this validation context was removed in commit 35c38c. It looks like, instead of relying on everyone validating the model to know when loa3 validation was important the validation always checks loa3 when it appears relevant.

```
git diff 35c38c^..35c38c app/models/user.rb | head -n21
diff --git a/app/models/user.rb b/app/models/user.rb
index afa63f067..be3ddb446 100644
--- a/app/models/user.rb
+++ b/app/models/user.rb
@@ -23,12 +23,12 @@ class User < Common::RedisStore
   validates :uuid, presence: true
 
   # conditionally validate if user is LOA3
-  with_options(on: :loa3_user) do |user|
-    user.validates :first_name, presence: true
-    user.validates :last_name, presence: true
-    user.validates :birth_date, presence: true
-    user.validates :ssn, presence: true, format: /\A\d{9}\z/
-    user.validates :gender, format: /\A(M|F)\z/, allow_blank: true
+  with_options if: :loa3? do |loa3_user|
+    loa3_user.validates :first_name, presence: true
+    loa3_user.validates :last_name, presence: true
+    loa3_user.validates :birth_date, presence: true
+    loa3_user.validates :ssn, presence: true, format: /\A\d{9}\z/
+    loa3_user.validates :gender, format: /\A(M|F)\z/, allow_blank: true
   end
```